contracts/Pair.sol

```
@@ -310,8 +321,10 @@ contract Pair is IPair, Initializable {
         int256 otherTokenQty = token == pair.tokenA.tokenAddress
             ? pair.tokenB.tokenQty
             : pair.tokenA.tokenQty;
-        int256 value = (spotTokenQty * precision) / otherTokenQty;
-
+        int256 value;
+        if (otherTokenQty > 0) {
+            value = (spotTokenQty * precision) / otherTokenQty;
```

otherTokenQty  == 0 then it will return 0 value which is wrong info, add a require check and let it fail

---

Actually UI can call getPairInfo() without adding the liquidity to read other values like 'getPrecisionValue','getFeePrecision',fee etc.

if we need to add revert condition then UI has to sure to call it only when pair having some quantities.